ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -284,7 +288,7 @@ final class SandboxServer(
         optTimeServiceBackend = timeServiceBackendO,
         metrics = metrics,
         healthChecks = healthChecks,
-        seedService = config.seeding.map(SeedService(_)),
+        seedService = Some(SeedService(config.seeding.getOrElse(SeedService.Seeding.Weak))),
```

Comment:
Does this need to be optional any more? I think this might be the only place that ever set it to `None`.

Reply:
Canton needs it for some days (hopfully).